tree/dataframe/src/RDFInterfaceUtils.cxx

```diff
@@ -340,16 +340,15 @@ std::vector<std::string> FindUsedColumnNames(std::string_view expression, const
    // To help matching the regex
```

Comment:
Are std::set and TPRegexp already #included?

Reply:
TPRegexp yes, set no: nice catch :)